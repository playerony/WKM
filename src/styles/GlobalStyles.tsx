import { createGlobalStyle } from 'styled-components'

const GlobalStyles = createGlobalStyle`
  html, body {
    color: ${({ theme }) => theme.color.gray};
    font-family: ${({ theme }) => theme.font.normal};
    background-color: ${({ theme }) => theme.color.cream};
  }
`

export default GlobalStyles
