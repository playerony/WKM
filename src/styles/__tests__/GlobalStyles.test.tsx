import React from 'react'
import renderer from 'react-test-renderer'

import GlobalStyles from '../GlobalStyles'

import snapshotTest from '@tests/snapshotTest'
import ThemeProviderMock from '@tests/ThemeProviderMock'

describe('GlobalStyles Component', () => {
  renderer.create(
    <ThemeProviderMock>
      <GlobalStyles />
    </ThemeProviderMock>
  )

  snapshotTest(document.head)
})
