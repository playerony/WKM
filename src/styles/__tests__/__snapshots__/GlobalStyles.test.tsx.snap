// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetStyles Component snapshot test should match 1`] = `
<head>
  <style
    data-styled="active"
    data-styled-version="5.2.1"
  >
    html,body{color:#707070;font-family:'HelveticaNeue',Helvetica,Arial,sans-serif;background-color:#16161a;}
  </style>
</head>
`;
